src/commons/shuffle/shuffle.js

```diff
@@ -36,6 +43,10 @@ export class Shuffle {
   ];
 
   handleSelect(product) {
+    if (product.name === 'marketing') {
```

Comment:
this call is emmited in navbar
`<shuffle select-product.call="setCurrentProduct(product)"></shuffle>`
and the same thing is done there:
```
setCurrentProduct({ url }) {
    if (!url) {
      return;
    }
    this.router.navigate(url);
  }
```